lib/ansible/modules/files/unarchive.py

```diff
@@ -334,10 +334,10 @@ def is_unarchived(self):
         if self.file_args['group']:
             try:
                 tgr = grp.getgrnam(self.file_args['group'])
-            except:
+            except KeyError:
```

Comment:
`grp.getgrnam` can also raise `ValueError` and `TypeError`

```
        self.assertRaises(TypeError, grp.getgrnam)
        self.assertRaises(ValueError, grp.getgrnam, 'a\x00b')
        self.assertRaises(KeyError, grp.getgrnam, fakename)
```

Although `TypeError` is likely not to occur here.

Reply:
Thanks. I will add `ValueError`.  
`TypeError` should only occur when `grp.getgrnam` is called without arguments, which is not the case here.